Package.swift

```diff
@@ -12,6 +12,7 @@ let package = Package(
 		.library(name: "Lexicon", targets: ["Lexicon"]),
 		.library(name: "SwiftLexicon", targets: ["SwiftLexicon"]),
 		.library(name: "SwiftStandAlone", targets: ["SwiftStandAlone"]),
+        .library(name: "KotlinStandAlone", targets: ["KotlinStandAlone"]),
```

Comment:
alignment seems wonky here, or is it GitHub?

Reply:
It's GitHub. When I checked it out it was fine.

Reply:
Same for me unsure wtf was going on

Reply:
This is because my entry uses spaces and rest uses tabs